src/main/kotlin/app/cash/licensee/task.kt

```diff
@@ -68,6 +68,9 @@ abstract class LicenseeTask : DefaultTask() {
   @get:Input
   internal abstract val unusedAction: Property<UnusedAction>
 
+  @get:Input
+  internal abstract val bundleAndroidAsset: Property<Boolean>
```

Comment:
AFAIK it's not needed. And can you add a test without calling licensee?

Reply:
Yeah looks unused

Reply:
For a test, we probably want to enable the flag, build an APK, and then maybe list its contents and look for the file? I'm not sure how thorough we really want to be here.

Reply:
Just run assemble to test the task wiring?

Reply:
Yeah that covers everything except the relative path within assets which is pretty good. Since I think we will already need a dedicated test function, adding a quick path check inside the APK shouldn't be much more work.

Reply:
Added the test case, let me know if its good.